chia/wallet/puzzles/cat.clvm

```diff
@@ -167,7 +167,10 @@
                     (r (r condition)))
          )
          (if (= (f condition) CREATE_COIN_ANNOUNCEMENT)
-           (list CREATE_COIN_ANNOUNCEMENT (sha256 0xca (f (r condition))))
+           (list CREATE_COIN_ANNOUNCEMENT
```

Comment:
I don't think this is correct. I think you need

`(c CREATE_COIN_ANNOUNCEMENT (sha256 0xca (f (r condition))) (r (r condition)))`